joomla/joomla-cms
# [4.3] Smart Search Sorting Drop-Down

### Summary of Changes
Adds a user defined sorting/ordering capability to the front-end of the Smart Search results.
It does that by adding a sorting drop-down above the results.

### Testing Instructions
1. Apply the PR and run `npm install` for the css to be generated in media.
2. Visit a Smart Search menu item.
2. Under the "Advanced" tab, enable the "Show Sort Fields".
3. Select the "Shown Sort Fields".
4. In the front-end search for a term that return results. You should see a sorting drop-down with the fields you set.
5. Play with the drop-down to change the sorting.

If your  "Sort Direction" is set to "Descending", you will NOT see the "Relevance ascending" option.
Seems like there is no reason to sort the results by "Relevance ascending".

### Actual result BEFORE applying this Pull Request
No front-end sorting.


### Expected result AFTER applying this Pull Request
Front-end sorting

![Smart-Search-sorting](https://user-images.githubusercontent.com/1838041/140910488-d6b9373f-a110-4b53-af40-8050fe8e3ca0.png)
### Documentation Changes Required
DKN
